src/components/cart/cart.tsx

```diff
@@ -1,28 +1,27 @@
-import classnames from 'classnames';
 import { useState } from 'react';
-import { useCart, useCartTotals } from '~/api/api-hooks';
+import { useCart, useCartTotals, useRemoveItemFromCart, useUpdateCartItemQuantity } from '~/api/api-hooks';
 import { useEcomAPI } from '~/api/ecom-api-context-provider';
 import { Drawer } from '~/components/drawer/drawer';
 import { isCartItemAvailable } from '~/utils';
-import { CartItem } from './cart-item/cart-item';
 import { useCartOpen } from './cart-open-context';
-import styles from './cart.module.scss';
+import { CartView } from './cart-view/cart-view';
 
 export const Cart = () => {
+    const ecomAPI = useEcomAPI();
     const { isOpen, setIsOpen } = useCartOpen();
     const { data: cart } = useCart();
     const { data: cartTotals } = useCartTotals();
-    const [checkoutAttempted, setCheckoutAttempted] = useState(false);
-    const ecomAPI = useEcomAPI();
+    const { trigger: updateQuantity } = useUpdateCartItemQuantity();
+    const { trigger: removeItem } = useRemoveItemFromCart();
+    const [error, setError] = useState<string>();
```

Comment:
is `error` a new functionality in this PR ?

Reply:
it was an experiment.. good catch

Reply:
or it was a test :)